ci/unit/test_main.sh

```diff
@@ -29,21 +29,14 @@ else
   BULLSEYE=
 fi
 
-# shellcheck disable=SC1091
-source ./.build_vars.sh
-rm -f "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/control"
-mkdir -p "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/"
-ln -s ../../../../../../../../src/control \
-  "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/control"
```

Comment:
This particular code was just wrong, not related to the unit test at all and occasionally failing.  This job shouldn't be touching SL_BUILD_DIR at all as that directory isn't valid on the node where this is run.